win_build.ps1

```diff
@@ -21,8 +21,8 @@ $commitHash = (git rev-parse HEAD)
 
 $exporterRepo = "github.com/prometheus-community/windows_exporter"
 $exporterBinaryName = "windows_exporter.exe"
-# Commit used by v0.16.0 of windows_exporter
-$exporterVersion = "1c199e6c0eed881fb09dfcc84eee191262215e5e"
+# Commit used by v0.25.1 of windows_exporter
+$exporterVersion = "f70fa009de541dc99ed210aa7e67c9550133ef02"
```

Comment:
The exporter has changed the way `collectors` gets register, so the current approach of removing the collectors files to only build the exporter with the required ones is no longer valid.